Working log_level and validate (#69)
openvinotoolkit/mediapipe: docs/development.md

'bazel-bin/mediapipe/examples/desktop/object_detection/object_detection_openvino --calculator_graph_config_file mediapipe/graphs/object_detection/object_detection_desktop_openvino_graph.pbtxt --input_side_packets "input_video_path=/mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4,output_video_path=/mediapipe/tested_video.mp4"'
```
- in object_detection example the input video will get all the objects detected by the model and return the tested_video.mp4 output.

# Setting log level

> Openvino Model Server has its own log_level setting that can be set to the following levels:

Reviewer: I think it should be clearer to state that the log level in mediapipe application can be set using GLOG_minloglevel env variable. `0` is the most verbose for debuging purposes and 3 is the least verbose with the major errors only.
That setting is applicable both for mediapipe framework logs and also for OpenVINO inference calculator.

    INFO, DEBUG, WARNING, TRACE, ERROR

Recommended development log_level setting is DEBUG as it will print out usefull information and perform additional OpenVINOInferenceCalculator calculator validation.
If you want to set the flag in the desktop examples applications you can do it by setting the GLOG_minloglevel environment variable.
If the GLOG_minloglevel variable is not set Model Server will use the default log_level - INFO.

| GLOG_minloglevel | Model Server log_level |
:------------------| :--------------------: |

> |                  |          INFO          |

Reviewer: why don't you use the order from highest to lowest level - TRACE, DEBUG,INFO,WARNING, ERROR ?

'bazel-bin/mediapipe/examples/desktop/object_detection/object_detection_openvino --calculator_graph_config_file mediapipe/graphs/object_detection/object_detection_desktop_openvino_graph.pbtxt --input_side_packets "input_video_path=/mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4,output_video_path=/mediapipe/tested_video.mp4"'
```
- in object_detection example the input video will get all the objects detected by the model and return the tested_video.mp4 output.

# Setting log level

The log level in mediapipe application can be set using GLOG_minloglevel env variable. 0 is the most verbose for debuging purposes and 3 is the least verbose with the major errors only.
That setting is applicable both for mediapipe framework logs and also for OpenVINO inference calculator.

> Openvino Model Server has its own log_level setting that can be set to the following levels:

Reviewer: it can not be set separately.

        dumpOvTensorInput(input,"input");
    #endif
```      
# Setting log level

The log level in mediapipe application can be set using GLOG_minloglevel env variable. 0 is the most verbose for debuging purposes and 3 is the least verbose with the major errors only.
That setting is applicable both for mediapipe framework logs and also for OpenVINO inference calculator.

> Openvino Model Server evaluates the GLOG_minloglevel value and sets its own log_level based on it to one of the following values:

Reviewer: ```suggestion
OpenVINO inference calculator log level is mapped to the defined GLOG_minloglevel value.
```

>     TRACE, DEBUG, INFO, ERROR

Reviewer: ```suggestion

```

> Recommended development log_level setting is DEBUG as it will print out usefull information and perform additional OpenVINOInferenceCalculator calculator validation.

Reviewer: ```suggestion
Recommended development GLOG_minloglevel setting is `0` as it will print most detailed information and perform additional graph and OpenVINOInferenceCalculator calculator validation.
```

If you want to set the flag in the desktop examples applications you can do it by setting the GLOG_minloglevel environment variable.

> If the GLOG_minloglevel variable is not set Model Server will use the default log_level - INFO.

Reviewer: ```suggestion

```

> | GLOG_minloglevel |      value       | Model Server log_level |

Reviewer: ```suggestion
| GLOG_minloglevel |      MediaPipe log level      | Model Server log_level |
```

:------------------|------------------| :--------------------: |

> |      INFO        |        0         |          TRACE         |

Reviewer: ```suggestion
|      0        |        INFO         |          TRACE         |
```

> |      WARNING     |        1         |          DEBUG         |

Reviewer: ```suggestion
|      1     |        WARNING         |          DEBUG         |
```

> |      ERROR       |        2         |          INFO          |

Reviewer: ```suggestion
|      2       |        ERROR         |          INFO          |
```

> |                  |      unset       |          INFO          |

Reviewer: ```suggestion
|                  |      ERROR       |          INFO          |
```

> |      FATAL       |        3         |          ERROR         |

Reviewer: ```suggestion
|      3       |        FATAL         |          ERROR         |
```
